api/src/routes/auth/validation.ts

```diff
@@ -43,10 +47,13 @@ export type Access = z.infer<typeof AccessSchema>;
 export const SignInSchema = z.object({
 	email: z
 		.string({ required_error: 'Email is required' })
-		.min(1, 'Email must be at least one character long')
-		.email('Invalid Email'),
+		.email('Invalid Email')
+		.trim()
+		.toLowerCase()
+		.min(1, 'Email must be at least one character long'),
```

Comment:
suggestion: trim before validating as email
```suggestion
		.trim()
		.toLowerCase()
		.min(1, 'Email must be at least one character long')
		.email('Invalid Email'),
```